pump/storage/log_test.go

```diff
@@ -73,6 +74,18 @@ func (lfs *LogFileSuit) TearDownTest(c *check.C) {
 	os.Remove(lfs.lf.path)
 }
 
+func (lfs *LogFileSuit) TestWriteOffset(c *check.C) {
+	dir := c.MkDir()
+	f, err := newLogFile(1024, filepath.Join(dir, "1024.vlog"))
+	c.Assert(err, check.IsNil)
+	c.Assert(f.GetWriteOffset(), check.Equals, int64(0))
+
+	data := make([]byte, 379)
+	err = f.Write(data, true)
+	c.Assert(err, check.IsNil)
+	c.Assert(f.GetWriteOffset(), check.Equals, int64(379))
```

Comment:
Is it a _write failed_ case (use `RLIMIT_FSIZE` as before) need?

Reply:
That case was already covered elsewhere.

Reply:
better to migrate the case here?

Reply:
Maybe leave that for another PR?

Reply:
are there something undefined and need to be handled separately in another PR?